lib/Dialect/FIRRTL/Transforms/CreateSiFiveMetadata.cpp

```diff
@@ -246,8 +248,13 @@ struct ObjectModelIR {
 
     auto memPaths = instancePathCache.getAbsolutePaths(mem);
     SmallVector<Value> memoryHierPaths;
+    SmallVector<Value> finalInstanceNames;
     for (auto memPath : memPaths) {
-      Operation *finalInst = memPath.leaf();
+      auto finalInst = memPath.leaf();
```

Comment:
nit, any reason to change this line? It might be a little smaller diff to keep the original `Operation *finalInst = ...`, and I find the type being spelled out helps personally.

Reply:
Need to get the instance name, updated the `Operation*` to `InstanceOpInterface`

Reply:
Ah I see, thanks.